### DISCUSSION TITLE
Launching Extension = black box

### DISCUSSION THREAD

**Discussion by coldstart01** [coddingtonbear/obsidian-web#188] [General]
How do i fix this ? I launch the extension and boom,
![Screenshot 2024-04-17 181143](https://github.com/coddingtonbear/obsidian-web/assets/50429954/49036d53-2a3e-417c-833b-1635d4952c5a)

**Comment by coddingtonbear** [comment]
I've never seen anything like that when just opening the extension -- could you reproduce the black box situation, then go into extension options (right-click on the extension icon) and do a bug report export from the upper-right corner of the screen?

I'm not sure of the context here -- are you just installing the extension and trying to open the dialog for the first time, or have you walked through the onboarding flow already?

**Reply by LightVolk** [reply]
I register plugin and work with it , but saddenly it happens. Now it works well (but it will be broke some days, this bug i see time to time). I load extension report on this bug

**Reply by LightVolk** [reply]
Sorry, i talk about this bug (same as this) - https://github.com/coddingtonbear/obsidian-web/discussions/190

**Comment by wscullen** [comment]
I encountered this as well. After trying to reinstall Local REST API and the web extension multiple times, nothing was working. When it finally started working the initial onboarding process started properly and then after that the extension started working properly. I'm not sure but it seems like the extension can get in a bad local state where none of the settings are retained, I wonder if chrome is keeping some of the local state for the extension between uninstall and install? I can't say for sure what triggered the onboarding process properly, just many tries of restarting obsidian, chrome, and reinstalling the browser extension. Another thing to try is to on the local REST side is regenerate all crypto, then download the certificate, add to the keychain, trust. Then try installing the web browser extension.

**Comment by dcsan** [comment]
I got this at first. there was no info about configuration. after going to `plugin > options` i saw the config options and then tried some stuff. I had to reload the current browser to get the local permissions to take hold.

**Reply by LightVolk** [reply]
Yes, it works on me -
1. remove plugin
2. close chrome
3. install plugin
4. load plugin template settings (with my notes patterns , require to create it at step 0)

**Comment by Tback1** [comment]
I sometimes encounter this phenomenon when I first launch the ob-web extension. But just need to refresh the web-page will solve it. I thought it was just that the extended resources hadn't been loaded yet, but it didn't happen consistently.
